packages/block-library/src/image/edit.js

```diff
@@ -481,15 +481,15 @@ class ImageEdit extends Component {
 									type="number"
 									className="block-library-image__dimensions__width"
 									label={ __( 'Width' ) }
-									value={ width !== undefined ? width : imageWidth }
+									value={ width !== undefined ? width || '' : imageWidth }
```

Comment:
Something I'm wondering here. If `width` is not undefined this means there's no need for `|| ''` in the first part of the ternary right? Is the warning triggered by the first part or the second part (without this PR)?

Reply:
Hi @youknowriad, 
Actually, when I tested this again, it looks like this ternary doesn't prevent the console warning.

https://github.com/WordPress/gutenberg/pull/15239/commits/282e052b40bd61700c4a5929d0a83989123678f7, which adds this ternary, looks to be a regression where the console warning appears again.

You made a good point in https://github.com/WordPress/gutenberg/pull/15239#discussion_r281590329 about the default value of `0` being strange. But I'm not sure how to prevent this console warning otherwise.

Thanks!